pkgs/development/python-modules/jax/default.nix

```diff
@@ -97,7 +97,10 @@ buildPythonPackage rec {
     "tests/"
   ];
 
-  disabledTestPaths = lib.optionals stdenv.hostPlatform.isDarwin [
+  disabledTestPaths = [
+    "tests/api_test.py::BackendsTest::test_no_backend_warning_on_cpu_if_platform_specified"
```

Comment:
This is not the correct syntax for skipping a test.
You can either add `"tests/api_test.py"` to `disabledTestPaths` if all tests in this file fail, or, add `test_no_backend_warning_on_cpu_if_platform_specified` to `disabledTests` to skip this very test.

Also, please add a comment above the line explaining why skipping this test is necessary.
You may also add the error message.

Reply:
Looks like the tests below follow the same pattern.  It's also documented in https://nixos.org/manual/nixpkgs/unstable/#using-pytestcheckhook.

I'll add a comment.